src/common/components/navbar/index.tsx

```diff
@@ -1,23 +1,23 @@
-import React, {Component} from "react";
+import React, { Component } from "react";
```

Comment:
I can't seem to find what exact line was changed due to linting changes, please revert to default linting if you can. @mghazanfar

Reply:
@feruzm I edited code for pencil button's placement and styling. I've disabled linting so wont be pushing these type of issues any more.